manifests/requirements.pp

```diff
@@ -101,7 +101,7 @@
     timeout     => $timeout,
     cwd         => $cwd,
     user        => $owner,
-    subscribe   => $subscribe,
+    subscribe   => $local_subscribe,
```

Comment:
Correct me if Im wrong, it sounds impossible to set subscribe when I want `$manage_requirements == false`.